lib/pharos/host/el7/scripts/upgrade-kubeadm.sh

```diff
@@ -8,7 +8,11 @@ if [ $(kubeadm version -o short) = "v${VERSION}" ]; then
     exit
 fi
 
-versionlock="/etc/yum/pluginconf.d/versionlock.list"
-linefromfile "^0:kubeadm-" $versionlock
-yum install -y "kubeadm-${VERSION}"
-lineinfile "^0:kubeadm-" "0:kubeadm-${VERSION}-0.*" $versionlock
\ No newline at end of file
+BIN_URL="https://dl.bintray.com/kontena/pharos-bin/kube/${VERSION}/kubeadm-${ARCH}.gz"
+
+curl -fsSL $BIN_URL -o /tmp/kubeadm.gz
+curl -fsSL "${BIN_URL}.asc" -o tmp/kubeadm.gz.asc
+gpg --verify /tmp/kubeadm.gz.asc /tmp/kubeadm.gz
+gunzip /tmp/kubeadm.gz
+install -o root -g root -m 0755 -t /usr/local/bin /tmp/kubeadm # XXX: overrides package version?
```

Comment:
This should be installed somewhere temporarily only for the duration of the upgrade, only until the kubeadm package can be upgraded... leaving behind a version of kubeadm in `/usr/local/bin` is bad.

Reply:
This also gets left behind on `pharos-cluster reset` and causes the next `pharos-cluster up` to run using the wrong version of kubeadm.